### PR TITLE
Move params section from base.config to nextflow.config

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/tools!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config [hunk 1]
```diff
@@ -24,7 +24,7 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = "./iGenomes"
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```
Comment (ewels) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: Woah, this has been in the template all along??

Comment (drpatelh) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: Yep.

Comment (ewels) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: 😱

Comment (drpatelh) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: `base.config` must have priority! 🤣

Comment (ewels) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: Yeah, I think it's loaded after this block.. Bit scary!

**Pull request merged**